src/request.rs

```diff
@@ -62,7 +63,8 @@ pub trait KvRequest: Request + Clone + Sync + Send + 'static + Sized {
                 let request = self.make_rpc_request(key_data, &store);
                 async move {
                     let request = request?;
-                    let response = store.dispatch::<_, Self::RpcResponse>(&request).await?;
+                    let response = store.dispatch::<_, Self::RpcResponse>(&request).await;
+                    let response = tikv_stats(request.label()).done(response)?;
```

Comment:
Should we create the stats before dispatch? Otherwise, the duration stats don't make much sense.

Reply:
Yes it should :-)